src/AdminSettings.vue

```diff
@@ -29,85 +29,73 @@
 
 		<ul class="password-policy__settings-list">
 			<li>
-				<input id="password-policy__settings__min-length"
-					v-model="config.minLength"
-					type="number"
-					@change="updateNumberSetting('minLength')">
-				<span>{{ t('password_policy', 'Minimum password length') }}</span>
+				<label>
+					<input id="password-policy__settings__min-length"
+						v-model="config.minLength"
+						type="number"
+						@change="updateNumberSetting('minLength')">
+					{{ t('password_policy', 'Minimum password length') }}
+				</label>
 			</li>
 			<li>
-				<input id="password-policy-history-size"
-					v-model="config.historySize"
-					type="number"
-					@change="updateNumberSetting('historySize')">
-				<span>{{ t('password_policy', 'User password history') }}</span>
+				<label>
+					<input id="password-policy-history-size"
+						v-model="config.historySize"
+						type="number"
+						@change="updateNumberSetting('historySize')">
+					{{ t('password_policy', 'User password history') }}
+				</label>
 			</li>
 			<li>
-				<input id="password-policy-expiration"
-					v-model="config.expiration"
-					type="number"
-					@change="updateNumberSetting('expiration')">
-				<span>{{ t('password_policy', 'Number of days until user password expires') }}</span>
+				<label>
+					<input id="password-policy-expiration"
+						v-model="config.expiration"
+						type="number"
+						@change="updateNumberSetting('expiration')">
+					{{ t('password_policy', 'Number of days until user password expires') }}
+				</label>
 			</li>
 			<li>
-				<input id="password-policy_failed-login"
-					v-model="config.maximumLoginAttempts"
-					type="number"
-					@change="updateNumberSetting('maximumLoginAttempts')">
-				<span>{{ t('password_policy', 'Number of login attempts before the user account is blocked (0 for no limit)') }}</span>
+				<label>
+					<input id="password-policy_failed-login"
+						v-model="config.maximumLoginAttempts"
+						type="number"
+						@change="updateNumberSetting('maximumLoginAttempts')">
+					{{ t('password_policy', 'Number of login attempts before the user account is blocked (0 for no limit)') }}
+				</label>
 			</li>
 		</ul>
+
 		<ul class="password-policy__settings-list">
-			<li />
 			<li>
-				<input id="password-policy__settings__enforce-non-common"
-					v-model="config.enforceNonCommonPassword"
-					type="checkbox"
-					class="checkbox"
-					@change="updateBoolSetting('enforceNonCommonPassword')">
-				<label for="password-policy__settings__enforce-non-common">
+				<CheckboxRadioSwitch :checked.sync="config.enforceNonCommonPassword"
```

Comment:
How about using type `switch`? Esp. as there is no save-feedback on these Checkboxes? 🤔

Reply:
Good idea!